functional-samples/sample.bookmarks/popup.js

```diff
@@ -51,7 +52,7 @@ function dumpNode(bookmarkNode, query) {
       chrome.tabs.create({ url: bookmarkNode.url });
     });
 
-    var span = $('<span>');
+    span = $('<span>');
```

Comment:
Why is this an empty string redefined? Is there a reason it can't just be a const?

Reply:
scope

Reply:
yeah, const was also my first try. eslint is better at these things than us humans

Reply:
🤖